app/test_main.py

```diff
@@ -1,3 +1,26 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_human_ages",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (29, 29, [3, 3]),
+        (100, 100, [21, 17])
+    ]
+)
+def test_should_return_correct_ages(
+        cat_age: int,
+        dog_age: int,
+        expected_human_ages: list[int]
+) -> None:
+
```

Comment:
Exceptions processing may include changes in the `test_should_return_correct_ages` function as well.